nuxt.config.ts

```diff
@@ -208,16 +208,34 @@ export default defineNuxtConfig({
         { rel: 'apple-touch-icon', href: '/apple-touch-icon.png' },
       ],
       meta: [
+        // iOS
+        { name: 'apple-mobile-web-app-capable', content: 'yes' },
+        { name: 'apple-mobile-web-app-title', content: 'Mozilla Social' },
         { name: 'apple-mobile-web-app-status-bar-style', content: 'black-translucent' },
+        // Android
+        { name: 'mobile-web-app-capable', content: 'yes' },
+        { name: 'theme-color', media: '(prefers-color-scheme: light)', content: 'white' },
+        { name: 'theme-color', media: '(prefers-color-scheme: dark)', content: 'black' },
+        // Windows
+        { name: 'msapplication-TileColor', content: '#5F22CB' },
+        { name: 'msapplication-TileImage', content: '/pwa-192x192.png' },
```

Comment:
I know I added this line, but we should probably update this `png` to be this file `https://assets.mozilla.social/site_uploads/files/000/000/001/@2x/ef49d116df000b31.png` 😅

Reply:
Isn't this used in the Windows Start Menu? I figured it made since to have the Live Tile image be square and simple?

Reply:
I could upgrade it to 512x512 ..